TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -506,12 +506,18 @@ public void DidChangeConfigurationParams(IEnumerable<string> arguments)
 
             var typeCobolOptions = new TypeCobolOptions(Configuration);
 
+            if (ClientConfigurationChangedEvent != null)
+                ClientConfigurationChangedEvent(this, arguments);
+
             //Configure CFG/DFA analyzer + external analyzers if any
-            var analyzerProvider = new CompositeAnalyzerProvider();
-            analyzerProvider.AddActivator((o, t) => CfgDfaAnalyzerFactory.CreateCfgAnalyzer("cfg-dfa", Configuration.CfgBuildingMode));
-            analyzerProvider.AddCustomProviders(Configuration.CustomAnalyzerFiles);
+            var compositeAnalyzerProvider = new CompositeAnalyzerProvider();
+            if (Configuration.CfgBuildingMode != CfgBuildingMode.None)
```

Comment:
no need to test here, the `CreateCfgAnalyzer` method returns null if building mode is set to None and the analyzer provider discards null values.